support/modules/file-install

```diff
@@ -46,6 +47,7 @@ case "$STATE" in
             filename=$(tar -tf ${update_files_tar})
             rm ${dest_dir}/${filename}
         else
+            test "$dest_dir" == "/" && exit 0
```

Comment:
Is the exit code correct here? Success?

Reply:
For the Roolback step yes, as there is nothing we can rollback to.

So what it will happen for an update with dest_dir `/` is that the ArtifactInstall will exit 1, then mender client will perform an rollback (a *successful* rollback), and then back to idle. The update will be marked as failed, though, but the client will stay in a consistent state.

Reply:
Thanks for explaining